### PR TITLE
Relax random provider

### PR DESCRIPTION
**Pull request by dor961**
Ran into provider dependency issues with other modules in the same project.

### CODE REVIEW

File: modules/postgresql/versions.tf [hunk 1]
```diff
@@ -19,6 +19,5 @@ terraform {
   required_providers {
     google = "~> 3.5"
     null   = "~> 2.1"
-    random = "~> 2.2"
```
Comment (morgante) [inline modules/postgresql/versions.tf:22]: You should at least pin a minimum version.

**Review by morgante — CHANGES_REQUESTED** [review]

File: modules/postgresql/versions.tf (hunk 1 above)
Comment (dor961) [reply modules/postgresql/versions.tf:22]: Could you please review the latest commit?
Thanks.

**Review by morgante — APPROVED** [review]

**Comment by billyfoss** [comment]
Is there any reason the null and random providers cannot use the 3.x version?  The breaking change in 3.x is dropping support for Terraform 0.11, which is not relevant to this module.

**Comment by morgante** [comment]
> Is there any reason the null and random providers cannot use the 3.x version? The breaking change in 3.x is dropping support for Terraform 0.11, which is not relevant to this module.

We should be fine to allow this.

**Comment by morgante** [comment]
This has been fixed.

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Pull request closed without merging**